tools/gened/upload_override_gened.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import csv
+import requests
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-e",
+    "--envs",
+    nargs="+",
+    choices=["production", "beta", "dev", "local"],
+    required=True,
+    help="environments to upload overrides to. Can be 'production', 'beta', 'dev' or 'local'. Can specify multiple environments.",
+)
+parser.add_argument(
+    "-f", "--file", required=True, help="csv override file to import from"
+)
+parser.add_argument(
+    "-t", "--token", required=True, help="admin token to authenticate with"
+)
+parser.add_argument(
+    "-p", "--port", required=False, default="3333", help="localhost port"
+)
+args = vars(parser.parse_args())
+
+envs: list[str] = args["envs"]
+sections_file: str = args["file"]
+admin_token: str = args["token"]
+local_port: str = args["port"]
+api_urls = {
+    "production": "https://cugetreg.com/_api/graphql",
+    "beta": "https://beta.cugetreg.com/_api/graphql",
+    "dev": "https://dev.cugetreg.com/_api/graphql",
+    "local": "http://localhost:"+local_port+"/_api/graphql",
+}
+for env in envs:
+    if env not in ["production", "beta", "dev", "local"]:
+        raise Exception("Invalid environment: " + env)
+    print("Uploading overrides to", api_urls[env])
+
+with open(sections_file, mode="r", encoding="utf-8-sig") as csvFile:

```

Comment:
Please write the comment for the structure of `sections_files` like

```suggestion
# section_files contains X columns as follows:
# courseName: the name of course in Chulalongkorn University
# courseNo: ...
# genEdTyp: ...
with open(sections_file, mode="r", encoding="utf-8-sig") as csvFile:
  # do things
```

Reply:
Also, wondering why not use `.ipynb` instead? For better data visualization and documentation

Reply:
idk, just didn't think of it.

Reply:
Also this will be deprecated after we have backoffice anyways

Reply:
FYI, Jupyter notebook has interactive control feature which allow you to use `input type=select`, `radio` and others. The blog that talking about this feature is here https://towardsdatascience.com/interactive-controls-for-jupyter-notebooks-f5c94829aee6